### PR TITLE
implementation of Container Image Tag Entity

### PR DESCRIPTION
**Pull request by tstrych**
@oshtaier please review.
#132 
[tests](https://github.com/SatelliteQE/robottelo/pull/6318)

### CODE REVIEW

File: airgun/views/containerimagetag.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    breadcrumb = BreadCrumb()
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+            breadcrumb_loaded
+            and self.breadcrumb.locations[0] == 'Container Image Tags'
+            and len(self.breadcrumb.locations) >= 2
```
[the remark this answers is not in the document]
Comment (tstrych) [reply airgun/views/containerimagetag.py]: @oshtaier details tab have breadcrumb ` len == 2 ` etc.
`Container Image Tags -> "name of instance"` 
but lifecycle environment have length 3 etc.
`Container Image Tags -> "name of instance" -> Lifecycle Environment `
What should I use in breadcrumb ?

**Comment by tstrych** [comment]
@oshtaier please review, 

and one question about breadcrumb 
details tab have breadcrumb `len == 2` etc.
`Container Image Tags -> "name of instance"`
but lifecycle environment tab have `len == 3` etc.
`Container Image Tags -> "name of instance" -> Lifecycle Environment`
What do you recommend here as proper usage of breadcrumb ?

**Comment by oshtaier** [comment]
`==2` as you get to Details page by default when you do your navigation

**Comment by abalakh** [comment]
@oshtaier can't agree here with you, should be `>=2` - `lce` tab is a part of details view as well.
Let's say user wants to read some entity and then assign lces. In case of `>=2` after read he'll be on details page (but len of breadcrumb will be 3) and will just click on lce or any other tab and proceed with association.
In case of `==2` it will navigate to entities list, search for specific entity and open it once more, despite the fact it's already opened.
We should only be careful with `more or equal` when there can be totally different views inside some entity tab (e.g. repositories inside products or filters inside content views) and specify `<=3` instead, but that's not the case here.

**Comment by tstrych** [comment]
@oshtaier sorry I just didn't notice. So thank you for reminding me. It should be done now.

**Pull request merged**